hydra-web/src/js/modules/datatable.js

```diff
@@ -921,6 +936,7 @@ function(_, $, dt, Backbone){
             localStorage[this.id+"_selected"]=JSON.stringify(this.selectedIds);
             var keys= _.keys(this.selectedIds), checkbox=this.views.selectable.find("span.checkbox");
             this.views.parent.find("#selectedCount").html("["+keys.length+"]");
+
```

Comment:
It looks like you first tried working in this section of code. I think that was right, and I have some advice for a potential better way to implement only enabling #hostFailFsDeadButton if one row is selected. Instead of a change to handleSelect,  I think a change to saveSelectedState is the proper way to handle it.

Specifically:
- Add a class to #hostFailFsDeadButton like "btn-show-one"
- Add handling in saveSelectedState for "btn-show-one" that removes class "disabled" if keys.length=1, and adds class "disabled" if keys.length != 1

I applaud your efforts in learning backbone and javascript. The code you've written probably works great, but I think the approach I'm suggesting will result in cleaner and more maintainable code in the long run.